### ISSUE TITLE
Algorithm selection

### ISSUE THREAD

**Issue by zolawuzhigang** [vmeta42/metaai#5]
For the battery failure AI prediction module corresponding to the meta42 project, what are the common AI algorithms? Which algorithms match with each other will make the final trained model more accurate and efficient？

**Comment by YachaoDong** [comment]
Question 1:
 I personally think there are two ideas: 1. Take it as a time series prediction problem: First predict the voltage, internal resistance and other key values in the next few days, these predicted values are labeled according to the fault judgment logic. If it exceeds the threshold, it can be considered as a fault label. Time series prediction methods include RNN, LSTM, deepAR, arima, Holt winters, Facebook prophet, etc.
2. Take it as a classification problem. Automatically label the data in combination with the actual production environment, and use classification algorithms such as random forest, gbdt, xgboost, lightgbm and so on.

Question 2: we need to constantly try to replace the algorithm in combination with the actual data, and there is no specific standard. The combination of good data and good algorithm will make the model prediction more accurate and efficient.
